src/modules/archipelago.ts

```diff
@@ -0,0 +1,1256 @@
+/// <reference path="moduleBase.ts" />
+
+
+class RCTRArchipelago extends ModuleBase {
+    FirstEntry(): void {//Loads on park starting for first time. Something in my code calls it as well
```

Comment:
The comment says you manually call this function from somewhere else, I couldn't find it though. But also calling this function directly would be confusing because the function is specifically named for its use,  if you need to do this you could move the meat of this function into an appropriately named function (or multiple) and call it from FirstEntry and wherever else you need to.
```suggestion
    FirstEntry(): void {//Loads on park starting for first time.
```